src/Informer/InfoAtCompile.cpp

```diff
@@ -12,3 +18,19 @@ std::string unit_test_build_path() noexcept {
 std::string unit_test_src_path() noexcept {
   return "@CMAKE_SOURCE_DIR@/tests/Unit/";
 }
+
+std::string info_from_build() noexcept {
+  static const std::string info = [] {
```

Comment:
This function does nothing except call the lambda and return it's result, so just remove the lambda and put the code in the function directly.

Reply:
Indeed. I wonder why it was coded like this in the first place..